src/main.tsx

```diff
@@ -106,9 +106,12 @@ async function getJQLResults(useSecondOrg: boolean = false) {
 
     const creds: string = btoa(`${user}:${token}`);
     const authHeader = getAuthHeader(useSecondOrg, token, user, creds, authType);
-    const jqlQuery = `https://${baseURL}/rest/api/${apiVersion}/search?jql=${settings?.jqlQuery}`;
 
-    const response = await axios.get(jqlQuery, {
+    // jql-query property if it exists at block level will override plugin settings.
+    const jqlQuery: string = block?.properties?.jiraJql ?? settings?.jqlQuery ?? '';
+    const jqlQueryURL = `https://${baseURL}/rest/api/${apiVersion}/search?jql=${encodeURIComponent(jqlQuery)}`;
```

Comment:
[encodeURIComopnent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) is added so that the query doesn't break -- it's being passed as a query string.